Civi/Civioffice/PhpWord/Util/DocxUtil.php

```diff
@@ -48,17 +48,24 @@ public static function combineRuns(string|array $xml): string|array {
       return array_map(__METHOD__, $xml);
     }
 
-    $rRegex = '<w:r(?: w:(?:rsidDel|rsidRPr|rsidR)="[^"]+")*>';
+    $rRegex = '<w:r(?: w:(rsidDel|rsidRPr|rsidR)="[^"]+")*>';
+    $tRegex = '<w:t(?: xml:space="[^"]*")?>';
```

Comment:
Is something like `<w:t xml:space>` (attribute without value and without empty value `=""`) valid or known to occur?